spec/factories/subjects.rb

```diff
@@ -21,9 +24,9 @@
       end
     end
 
-    factory :migrated_project_subject do
-      locations({standard: "http://www.galaxyzoo.org.s3.amazonaws.com/subjects/standard/1237679543502373086.jpg"})
-      migrated true
-    end
+    #factory :migrated_project_subject do
```

Comment:
We can support them with the external_url in Medium, but they don't require and special handling or tests from the subject model.